crates/cxx-qt-lib/src/core/qdate.rs

```diff
@@ -86,3 +223,98 @@ unsafe impl ExternType for QDate {
     type Id = type_id!("QDate");
     type Kind = cxx::kind::Trivial;
 }
+
+#[cfg(feature = "chrono")]
+use chrono::Datelike;
+
+#[cfg(feature = "chrono")]
+impl From<chrono::NaiveDate> for QDate {
```

Comment:
I've gone with by value in this pull, elsewhere we thought it was more typical in Rust to by value. We should probably other API both internal and external to figure out if they should be by value or ref?

Eg here should it be as we end up copying internally anyway ?